dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/deduplication/hibernate/HibernatePotentialDuplicateStore.java

```diff
@@ -225,7 +227,7 @@ public void moveRelationships( TrackedEntityInstance original, TrackedEntityInst
 
     @Override
     public void moveEnrollments( TrackedEntityInstance original, TrackedEntityInstance duplicate,
-        List<String> enrollments )
+        List<String> enrollments, Date mergeDate )
```

Comment:
I don't think it's important that we send the exact same date in both enrollment and tei; I think it would just be better to create two new Date() and remove the Date from the method signature :)

Reply:
we have to ask @vilkg if it is required all the process has same update date

Reply:
I think it's okay if the date is not exactly the same. :)